docs/v3/release-notes/oss/version-3-6.mdx

```diff
@@ -2,6 +2,65 @@
 title: 3.6
 ---
 
+## 3.6.5
+
+*Released on December 04, 2025*
+
+**New Features 🎉**
+
+* Support dot-delimited syntax for job_variables in workers by [@devin-ai-integration](https://github.com/devin-ai-integration)[bot] in [#19619](https://github.com/PrefectHQ/prefect/pull/19619)
+
+**Enhancements ➕➕**
+
+* Raise if event is too big by [@jakekaplan](https://github.com/jakekaplan) in [#19531](https://github.com/PrefectHQ/prefect/pull/19531)
+* Fix anyio fail_after usage - async with was removed in recent version by [@saschwartz](https://github.com/saschwartz) in [#19583](https://github.com/PrefectHQ/prefect/pull/19583)
+* Unify definition of EventBeingProcessed type in prefect redis by [@saschwartz](https://github.com/saschwartz) in [#19585](https://github.com/PrefectHQ/prefect/pull/19585)
+* Support Path objects in result_storage parameter by [@zzstoatzz](https://github.com/zzstoatzz) in [#19533](https://github.com/PrefectHQ/prefect/pull/19533)
+* Hide "Join the Community" in air-gapped deployments by [@zzstoatzz](https://github.com/zzstoatzz) in [#19540](https://github.com/PrefectHQ/prefect/pull/19540)
+* Parametrize xreadgroup batch size in consumer and add EventPersister service setting by [@saschwartz](https://github.com/saschwartz) in [#19545](https://github.com/PrefectHQ/prefect/pull/19545)
+* Parametrize xautoclaim batch size by [@saschwartz](https://github.com/saschwartz) in [#19557](https://github.com/PrefectHQ/prefect/pull/19557)
+* Parametrize TaskRunRecorder and Trigger service message read batch size by [@saschwartz](https://github.com/saschwartz) in [#19559](https://github.com/PrefectHQ/prefect/pull/19559)
+* Add Slack GovCloud (slack-gov.com) webhook support to SlackWebhook block by [@devin-ai-integration](https://github.com/devin-ai-integration)[bot] in [#19565](https://github.com/PrefectHQ/prefect/pull/19565)
+* Add client-side caching for concurrency limits by [@devin-ai-integration](https://github.com/devin-ai-integration)[bot] in [#19625](https://github.com/PrefectHQ/prefect/pull/19625)
+
+**Bug Fixes 🐞**
+
+* Support username:password auth for self-hosted BitBucket instances by [@zzstoatzz](https://github.com/zzstoatzz) in [#19516](https://github.com/PrefectHQ/prefect/pull/19516)
+* Use parameterized queries in Snowflake SPCS worker by [@ColeMurray](https://github.com/ColeMurray) in [#19507](https://github.com/PrefectHQ/prefect/pull/19507)
+* Fix docker compose file in self hosted docs by [@rodeyfeld](https://github.com/rodeyfeld) in [#19551](https://github.com/PrefectHQ/prefect/pull/19551)
+* Optimize /api/task_runs/count endpoint performance by [@naaa760](https://github.com/naaa760) in [#19548](https://github.com/PrefectHQ/prefect/pull/19548)
+* Fix task caching bug in get_dbt_cloud_run_info for polling loops by [@devin-ai-integration](https://github.com/devin-ai-integration)[bot] in [#19597](https://github.com/PrefectHQ/prefect/pull/19597)
```

Comment:
This fix was made in an integration and can be excluded from these release notes.
```suggestion
```